app/grandchallenge/workstations/templatetags/workstations.py

```diff
@@ -79,3 +80,27 @@ def workstation_query(  # noqa: C901
         )
 
     return urlencode(query)
+
+
+@register.simple_tag()
+def workstation_query(**kwargs):
+    return get_workstation_query_string(**kwargs)
+
+
+@register.simple_tag()
+def workstation_session_control_data(
+    workstation=None, context_object=None, timeout=None, **kwargs
```

Comment:
This is quite a complex method so enforce kwargs only.

```suggestion
    *, workstation=None, context_object=None, timeout=None, **kwargs
```